torch/_C/__init__.pyi.in

```diff
@@ -226,3 +229,29 @@ class _CudaDeviceProperties:
     total_memory: _int
     is_integrated: _int
     is_multi_gpu_board: _int
+
+# Defined in torch/csrs/cuda/Stream.cpp
```

Comment:
Very minor: typo, should be `csrc`, both here and in comment further down.

Reply:
@rgommers, thank you very much for the thorough review. Updated.